source/time.tex

```diff
@@ -10764,6 +10769,7 @@
 If the \fmtgrammarterm{chrono-specs} is omitted,
 the chrono object is formatted
 as if by streaming it to \tcode{std::ostringstream os}
+with the formatting locale imbued
```

Comment:
Problem with paper/spec: What does it mean for a locale to be "imbued"? Same for uses of "imbued" elsewhere. Should we open an issue for this?

Reply:
iostreams have a member function "imbue" that attaches a locale to the stream. That's what's discussed here.